src/gui/shellwidget/shellwidget.cpp

```diff
@@ -153,6 +153,7 @@ void ShellWidget::paintEvent(QPaintEvent *ev)
 
 					if (cell.bold || cell.italic) {
 						QFont f = p.font();
+						f.setStyleName(QStringLiteral(""));
```

Comment:
Can we put a comment above this line

```
// #575: Clear style name. The KDE/Plasma theme plugin may set this
// but we want to match the family name with the bold/italic attributes.
```

Reply:
I don't think `QStringLiteral()` adds anything here? It should be converted to QString by the compiler anyways.

Just `""` should suffice :)